Sources/Parsers/PlistParser.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlistParser.swift
+//  SwiftGenKit
+//
+//  Created by Toshihiro Suzuki on 4/12/17.
+//  Copyright © 2017 SwiftGen. All rights reserved.
+//
+
+import Foundation
+import PathKit
+
+public final class PlistParser {
+
+  var keys: [String] = []
```

Comment:
Thanks for the review!
I completely agree with you, but I think I should do the fix in different PR.
This PR is enough for introducing `plist` command, at minimum. We can always improve in next release, supporting type inference and multiple files.